Makefile

```diff
@@ -131,7 +131,7 @@ ifeq (, $(shell which kustomize))
 	KUSTOMIZE_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$KUSTOMIZE_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/kustomize/kustomize/v3@v3.8.7 ;\
+	go install sigs.k8s.io/kustomize/kustomize/v3@v3.8.7 ;\
```

Comment:
@thearifismail this does not seem to work (for this version):
```
go: sigs.k8s.io/kustomize/kustomize/v3@v3.8.7 (in sigs.k8s.io/kustomize/kustomize/v3@v3.8.7):
	The go.mod file for the module providing named packages contains one or
	more exclude directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```

Reply:
@vkrizan ah - I've been using a version of kustomize installed in /usr/loca/bin. I just pushed a fix to master. Try with the new version.

Reply:
thanks!